app/models/template_input.rb

```diff
@@ -91,11 +91,12 @@ def resolved_value
 
   class UserInputResolver < InputResolver
     def ready?
-      !input_value.nil?
+      @renderer.invocation
     end
 
     def resolved_value
-      input_value.value
+      raise(ValueNotReady, _("Required input not specified")) if @input.required? && input_value.try(:value).blank?
```

Comment:
ok, I have just probably a different view on what's considered some value, e.g. when we have default value set to '-y' and you'd want to disable it I still see emptry string as valid value...

nevermind :-) we'll see if it causes any issue, definitely not a blocker